src/core/ext/transport/cronet/BUILD

```diff
@@ -0,0 +1,74 @@
+# gRPC Bazel BUILD file.
+#
+# Copyright 2016 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bazel:grpc_build_system.bzl",
+    "grpc_cc_library",
+)
+
+licenses(["notice"])  # Apache v2
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "-layering_check",
+        "-parse_headers",
+    ],
+)
+
+grpc_cc_library(
+    name = "grpc_transport_cronet_client_secure",
+    srcs = [
+        "client/secure/cronet_channel_create.cc",
+        "transport/cronet_api_dummy.cc",
+        "transport/cronet_transport.cc",
+        "transport/cronet_transport.h",
```

Comment:
`.h` should be in `hdrs` section

Reply:
It sounds counter-intuitive but srcs can include header files which are supposed to be private. gRPC seems not to distinguish private and public headers so far, though.